.github/workflows/check-lua.yml

```diff
@@ -26,6 +28,10 @@ jobs:
             --dest rts/Lua/library/generated \
             --repo https://github.com/${{ github.repository }}/blob/${{ github.sha }} \
             --error
+          lua-doc-extractor rts/Rml/SolLua/*.cpp \
```

Comment:
Just add as an argument instead of a new execution e.g. `lua-doc-extractor foo bar` instead of `lua-doc-extractor foo \n lua-doc-extractor bla`. Same for other instances below. cc @rhys-vdw thoughts?